plugins/modules/network_settings_workflow_manager.py

```diff
@@ -347,15 +347,42 @@
                     description: Retain existing banner message.
                     type: bool
                 type: dict
+              wired_data_collection:
+                description: Enables or disables the collection of data from wired network devices for telemetry and monitoring purposes.
+                    Applicable from sdk version 2.3.7.6 onwards.
+                suboptions:
+                    enable_wired_data_collection:
+                      description: Enable or disable wired data collection.
+                      type: bool
+                      default: false
+                type: dict
+              wireless_telemetry:
```

Comment:
Can't we say wireless_data_collection?
OR
wired_telemetry
wireless_telemetry 
?

Reply:
In wired_data_collection, "data collection" here refers to gathering specific identity information (like MAC and IP addresses) from wired endpoints in the network. 

In Wireless_telemetry, Telemetry, in the context of wireless networks, refers to the continuous streaming and monitoring of performance-related data from wireless devices (e.g., wireless controllers, access points, wireless clients). 
This data provides real-time insights into the health of wireless devices and networks, which may include metrics like signal strength, connection stability etc.
For these above mentioned reasons we have used Wireless_telemetry instead of wired_data_collection because it emphasises the real-time streaming and monitoring aspect, rather than just collection. 
if still any changes needed, please let me know, thank you.